src/test/database/removeRemote.spec.ts

```diff
@@ -32,7 +34,7 @@ describe("RemoveRemote", () => {
         x: true,
         f: true,
       });
-    return expect(remote.listPath("/", 1234)).to.eventually.eql(["a", "x", "f"]);
+    return expect(listRemote.listPath("/", 1234)).to.eventually.eql(["a", "x", "f"]);
```

Comment:
Is this the same test as `ListRemote` above? If so, remove it as testing `listRemote` in the `removeRemote` block is not correct (and is redundant)

Reply:
This test is redundant, removed in the next commit.